WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -393,6 +396,18 @@ public static int getThemeImageSizeWidth() {
         }
     }
 
+    // Wizard-style login flow
```

Comment:
The idea is to mirror how other flags operate and allow it to be activated on-purpose via a magic URL. The deeplink receiver is not there yet though so, unless we can keep it as is for now, I can add the receiver for completeness, or I can remove the pref and add it in a future PR. WDYT?